lib/big_query/rollup/hourly_downloads.ex

```diff
@@ -23,13 +25,19 @@ defmodule BigQuery.Rollup.HourlyDownloads do
     """
   end
 
-  defp format_results({rows, meta}, day) do
-    {Enum.map(rows, &(format_result(&1, day))), meta}
+  defp format_results(rows, from) do
+    epoch = Timex.beginning_of_day(from) |> Timex.to_unix()
+    Enum.map(rows, &(format_result(&1, epoch)))
   end
 
-  defp format_result(%{hour: hour} = row, day) do
+  defp format_result(%{hour: hour} = row, epoch) do
     row
-    |> Map.put(:dtim, Timex.shift(day, hours: hour))
```

Comment:
This was a bit odd ... as I was testing this out, I discovered that [it's real slow](https://github.com/bitwalker/timex/issues/357) to shift timezone-aware timestamps.  Just doing the epoch match is > 10x faster.